onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -2022,7 +2029,25 @@ including arg name, arg type (contains both type and shape).)pbdoc")
               const std::map<std::string, const py::object>& pyfeeds, RunOptions* run_options = nullptr)
                -> py::list {
              NameMLValMap feeds;
-             feeds.reserve(pyfeeds.size());
+             if (run_options != nullptr && !run_options->active_adapters.empty()) {
```

Comment:
maybe also add a warning about ignoring any active adapters in run options for the other run methods.

Reply:
also `run_with_iobinding`